playbooks/roles/edx_django_service/defaults/main.yml

```
@@ -104,6 +104,7 @@ edx_django_service_basic_auth_exempted_paths: '{{ edx_django_service_basic_auth_
 edx_django_service_newrelic_appname: '{{ COMMON_ENVIRONMENT }}-{{ COMMON_DEPLOYMENT }}-{{ edx_django_service_name }}'
 edx_django_service_enable_newrelic_distributed_tracing: false
 edx_django_datadog_service: 'edx-{{ edx_django_service_name }}'
+edx_specifically_enabled_datadog_apps: '{{ COMMON_ENABLE_SPECIFIC_DATADOG_APPS }}'
```

If you lift the logic up to this as a default variable, then you can just use `{{ edx_django_service_datadog_enable }}` in the tasks (avoiding repetition of the logic).

```suggestion
edx_django_service_datadog_enable: '{{ COMMON_ENABLE_DATADOG and (COMMON_ENABLE_DATADOG_APP or edx_django_service_name in COMMON_ENABLE_DATADOG_APP_SERVICES }}'
```

---

Here's another place you'd need to use that value: https://github.com/edx/configuration/blob/01d773fbd5efaa0cd4616093794d1b8a25b84e65/playbooks/roles/edx_django_service/templates/edx/app/app/app.sh.j2#L23